### DISCUSSION TITLE
Project naming: BOS Web Engine vs BOS Web Engine 2

### DISCUSSION THREAD

**Discussion by mpeterdev** [near/react-on-chain#5] [Announcements]
There is some inconsistency in the way we currently refer to this effort that could lead to confusion in the community. BOS Web Engine, BOS Web Engine 2, VM 2, etc

There are two ways we could look at it:
1. this project is the BOS Web Engine and it is aimed at replacing the previous rendering layer which we refer to as the VM or Discovery VM. All notion of v1 and v2 are dropped
2. this is the BOS Web Engine 2 and the current VM starts to be referred to as BOS Web Engine 1

I am in favor of <span>#</span>1, since the current rendering layer is already commonly called the VM and this project is a totally different approach as opposed to and further revision

**Comment by charleslavon** [comment]
+1 for `#1`
